Kernel/System/MigrateFromOTRS/CloneDB/Driver/mysql.pm

```diff
@@ -288,15 +289,20 @@ sub AlterTableAddColumn {
     my $SQL         = qq{ALTER TABLE $Param{Table} ADD $Param{Column} $ColumnInfos{DATA_TYPE}};
 
     if ( $ColumnInfos{LENGTH} ) {
-        $SQL .= " \($ColumnInfos{LENGTH}\)";
+        $SQL .= " ($ColumnInfos{LENGTH})";
     }
 
```

Comment:
I think that $ColumnInfos{COLUMN_DEFAULT} should be quoted, just in case it contains a single quote character.